docs/guidesdraft/01-getting-started.md

```diff
@@ -0,0 +1,228 @@
+
+### Previous requirements
+
+There are no previous requirements to complete this guide, but a basic knowledge of HTML, CSS and JavaScript would be helpful. In order to start creating maps, you will need text editor and an Internet connection.
+
+### What is CARTO VL?
+
+[CARTO VL](https://github.com/cartodb/carto-vl) is a JavaScript library to create custom location intelligence applications that leverage the power of [CARTO](https://carto.com/). It uses [WebGL](https://www.khronos.org/webgl/) to enable powerful vector maps.
+
+It relies on [Mapbox GL](https://www.mapbox.com/mapbox-gl-js/api) to render the basemaps. Mapbox GL can be used for other functionality too, read [Mapbox GL documentation](https://www.mapbox.com/mapbox-gl-js/api/) for more information. However, keep in mind that CARTO VL layers can only be controlled with the [CARTO VL API]() and that Mapbox GL native layers can **only** be controlled with the Mapbox GL API. Therefore, CARTO VL expressions cannot be used for Mapbox GL layers and vice versa.
+
+### Basic setup
+
+<div class="example-map">
+    <iframe 
+        id="getting-started-basic-styling-map"
+        src="https://carto.com/developers/carto-vl/examples/maps/guides/getting-started/basic-styling.html" 
+        width="100%"
+        height="500" 
+        frameBorder="0">
+    </iframe>
+</div>
+
+```html
+<head>
+  <!-- Include CARTO VL JS -->
+  <script src="https://libs.cartocdn.com/carto-vl/%VERSION%/carto-vl.js"></script>
+  <!-- Include Mapbox GL JS -->
+  <script src="https://libs.cartocdn.com/mapbox-gl/v0.48.0-carto1/mapbox-gl.js"></script>
+  <!-- Include Mapbox GL CSS -->
+  <link href="https://libs.cartocdn.com/mapbox-gl/v0.48.0-carto1/mapbox-gl.css" rel="stylesheet" />
+</head>
+```
+
+The easiest way to use CARTO VL is to include the required files from our CDN as you see in the example above: `https://libs.cartocdn.com/carto-vl/%VERSION%/carto-vl.js`. If you prefer to use the minified file, which is lighter and therefore loads faster, you can use `https://libs.cartocdn.com/carto-vl/%VERSION%/carto-vl.min.js`.
+
+Then, you will need to add Mapbox GL JavaScript and CSS files. This will let you use both `carto` and `mapboxgl` in your code.
+
+> Currently, not every Mapbox GL version is compatible with CARTO VL. We highly recommend the following combination. If you are importing CARTO VL from npm, you have to use our Mapbox GL fork. Read more about how to do this in [the advanced guide](https://carto.com/developers/carto-vl/guides/advanced)
+
+Create a `div` where the map is going to be drawn.
+
+```html
+  <div id="map"></div>
+```
+
+Style the `div` to ensure it will be displayed correctly.
+
+```css
+#map {
+    position: absolute;
+    height: 100%;
+    width: 100%;
+}
+```
+
+#### Defining the basemap
```

Comment:
Change all the headers to follow the same structure as in the examples https://github.com/CartoDB/carto-vl/issues/966